scripts/create_dm_and_gm.go

```diff
@@ -0,0 +1,55 @@
+package scripts
+
+import (
+	"github.com/mattermost/mattermost-server/v6/model"
+	"go.uber.org/zap"
+
+	"github.com/Brightscout/mattermost-load-test-scripts/constants"
+	"github.com/Brightscout/mattermost-load-test-scripts/serializers"
+	"github.com/Brightscout/mattermost-load-test-scripts/utils"
+)
+
+func CreateDMAndGM(config *serializers.Config, logger *zap.Logger) error {
+	client := model.NewAPIv4Client(config.ConnectionConfiguration.ServerURL)
+	if _, _, err := client.Login(config.ConnectionConfiguration.AdminEmail, config.ConnectionConfiguration.AdminPassword); err != nil {
```

Comment:
same

Reply:
this is removed in the new changes https://github.com/Brightscout/mattermost-load-test-scripts/blob/9eb17b6c8f0c02b038a1b8cbb5ee707145b165e9/scripts/create_dm_and_gm.go#L14